Tools/CISupport/ews-build/steps.py

```diff
@@ -4926,25 +5003,32 @@ def evaluateCommand(self, cmd):
         self.setProperty('build_summary', commit_summary)
         self.addURL(self.identifier, self.url_for_identifier(self.identifier))
 
-        return rc
+        defer.returnValue(rc)
 
     def url_for_hash_details(self, hash):
         return '{}{}/json'.format(COMMITS_INFO_URL, hash)
 
     def url_for_identifier(self, identifier):
         return '{}{}'.format(COMMITS_INFO_URL, identifier)
 
+    @defer.inlineCallbacks
     def identifier_for_hash(self, hash):
         try:
-            response = requests.get(self.url_for_hash_details(hash), timeout=60)
+            response = yield TwistedAdditions.request(
+                url=f'{COMMITS_INFO_URL}{hash}/json', logger=print,
```

Comment:
Is `url_for_hash_details` used elsewhere since we just removed its usage here.

Reply:
Good catch, removing.